client/app/services/policy/AbstractPolicy.js

```diff
@@ -0,0 +1,56 @@
+export default class AbstractPolicy {
```

Comment:
We have a single use case for the Policy object and that we override/extend it in the SaaS version. The SaaSPolicy object inherits from Policy (now DefaultPolicy). I don't see any future case we will ever bother extending AbstractPolicy...